app/views/application_letters/_form.html.erb

```diff
@@ -84,7 +91,7 @@
 
   <div class="form-group">
     <div class="col-lg-offset-2 col-lg-10">
-      <%= f.submit nil, :class => 'btn btn-primary' %>
+      <%= f.submit nil, :class => 'btn btn-primary', :disabled => @application_letter.after_deadline? %>
```

Comment:
A user could simply remove the class and submit the form. A server-side check is needed!